modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/test/PublicationUserNotificationHandlerTest.java

```diff
@@ -80,6 +84,11 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@After
+	public void tearDown() throws Exception {
+		ServiceContextThreadLocal.popServiceContext();
+	}
+
```

Comment:
If you're popping the service context in a tearDown, you should do the push in the setUp and assign the service context object to a class var. 

Another option is to do it locally in each tests with a try-finally bock.